### ISSUE TITLE
Last web-ifc implementation is not compatible with the last web-ifc-three implementation

### ISSUE THREAD

**Issue by jespa007** [IFCjs/web-ifc-three#149]
Hi there,

If I try to build current implementation of web-ifc-three (i.e 0.0.122) with the lastest implementation of web-ifc, when it tries to compile/transpile it throws the following errors,

![image](https://user-images.githubusercontent.com/7852409/217299264-2e97209d-45e8-4d95-8d26-0273cb671b2d.png)

So web-ifc-three it has to be adapted with the latest changes of web-ifc, because some signatures were changed since web-ifc 0.0.36

**Comment by jespa007** [comment]
@agviegas , I thing I've achieved make compatible last implementation web-ifc with last implementation of web-ifc-three. I'll make PR in a while.

**Comment by jespa007** [comment]
@agviegas , I think it has no sense do a PR that makes compatibles to the last web-ifc, because people will still use web-ifc from npm and to build web-ifc, copy files is not easy task. But anyway, tell me if you are interested or not still in do the PR

**Comment by beachtom** [comment]
So there have been a few API changes to support multi-schema in the new web-ifc. My suggestion is wait for us to get the new version of web-ifc done then adapt web-ifc-three

**Comment by jespa007** [comment]
> So there have been a few API changes to support multi-schema in the new web-ifc. My suggestion is wait for us to get the new version of web-ifc done then adapt web-ifc-three

I agree with that @beachtom

**Comment by agviegas** [comment]
Agreed! Both web-ifc-three and web-ifc-viewer need a big update / clean up, so this might be a good opportunity.

**Comment by olange** [comment]
I opened IFCjs/web-ifc-viewer#188 a few minutes ago and discovered this one afterwards. I guess both are intermingled. This comment to cross-reference both.
